yang/frr-route-map.yang

```diff
@@ -46,43 +49,54 @@ module frr-route-map {
      OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.";
 
   revision 2019-07-01 {
-    description "Initial revision";
+    description
+      "Initial revision";
   }
 
   /*
    * Types.
    */
   typedef route-map-sequence {
-    description "Route map valid sequence numbers";
     type uint16 {
       range "1..65535";
     }
+    description
+      "Route map valid sequence numbers";
```

Comment:
nit : numbers.
Dot symbol must be used at the end of sentences. Applicable for other places too.

Reply:
Added "." for all descriptions of the fields.